TelCo.ColorCoder/PairNumtoColor.cs

```diff
@@ -0,0 +1,42 @@
+using System;
+using System.Diagnostics;
+using System.Drawing;
+
+namespace TelCo.ColorCoder
+{
+ partial class Program
```

Comment:
Any reason for all this functionality to be in a single class called Program? The class name needs to reflect its responsibility.